documentation/configuringDynaflowLauncher/DynaflowLaunchersInputFiles.tex

```diff
@@ -159,9 +161,10 @@ \subsection{Configuration file}
 \caption{Chosen Outputs depending on the buildtype and the simulation kind : outputs chosen by default are \textcolor{green}{green} and optional inputs are \textcolor{gray}{gray}}
 \end{table}
 
-The systematic analysis parameters are described below:
+The \textit{StartingPointMode} indicates the starting state of the simulation. If it's warm, the simulation starts from a previous simulation state. If it's flat, the simulation starts from scratch.
```

Comment:
\begin{itemize}
\item \textbf{warm}: we assume that the initial $P$, $V$ and $\theta$ were already estimated and are available in the network description file for each node;
\item \textbf{flat}: we assume that node state is not available in the network description file
\end{itemize}

what do you think @marcochiaramello ?